### PR TITLE
Add Task1

### PR DESCRIPTION
**Pull request by EugeneDenkevich**
I added in task1 two solutions of our current task.

### CODE REVIEW

File: Tasks/Denkevich/Task1/Task1.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# --------------------------
+
+sum = input('Enter the sum of the deposit: \n')
+t = input('Enter the deposit term: \n')
+per = input('Enter the percent: \n')
+val = input('Would you like to capitalize your deposit monthly? (y/n): \n')
+
+sum = int(sum)
+t = int(t)
+per = int(per)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Denkevich/Task1/Task1.py:22]: By adapting these values for the **int** type and using them in the math expression you get the **float** number as answer, which tends to accumulate some error, it's better to use **decimal** for money-related operations, -10 points for the logical mistake

File: Tasks/Denkevich/Task1/Task1.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+if (val == 'y'):
+    D = sum * math.pow((1 + (per/100)/12), (t*12))
+    print('Calculated value of the deposit is: %.2f BYN' % D)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Denkevich/Task1/Task1.py:26]: old way of the string formatting, try not to use it anymore, the community has moved away from it long time ago

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, the final score is 90

**Comment by EugeneDenkevich** [comment]
Ok, thank you. =)
Can i correct my homework by using decimal instead int? Or it's the final version? I realy want to get the best mark! =)

>Воскресенье, 17 июля 2022, 8:48 +03:00 от MikitaTsiarentsyeu ***@***.***>:
>
***@***.*** approved this pull request.
>Good job, the final score is 90
>----------------------------------------------------------------------
>In  Tasks/Denkevich/Task1/Task1.py :
>> +sum = int(sum)
>+t = int(t)
>+per = int(per)
>
>By adapting these values for the  int type and using them in the math expression you get the  float number as answer, which tends to accumulate some error, it's better to use  decimal for money-related operations, -10 points for the logical mistake
>----------------------------------------------------------------------
>In  Tasks/Denkevich/Task1/Task1.py :
>> +
>+# With entering data and importing math:
>+# --------------------------
>+
>+sum = input('Enter the sum of the deposit: \n')
>+t = input('Enter the deposit term: \n')
>+per = input('Enter the percent: \n')
>+val = input('Would you like to capitalize your deposit monthly? (y/n): \n')
>+
>+sum = int(sum)
>+t = int(t)
>+per = int(per)
>+
>+if (val == 'y'):
>+    D = sum * math.pow((1 + (per/100)/12), (t*12))
>+    print('Calculated value of the deposit is: %.2f BYN' % D)
>
>old way of the string formatting, try not to use it anymore, the community has moved away from it long time ago
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you authored the thread. Message ID:  <MikitaTsiarentsyeu/Md-PT1-52-22/pull/25/review/1041072706 @ github . com>

С уважением,
Евгений Иванов
***@***.***

**Pull request merged**